keyboards/cest73/tkm/config.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xBADD
+#define PRODUCT_ID      0xAA55
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    cest73
+#define PRODUCT         "Ten Key Mirrored 2021 Release 1.03"
```

Comment:
Manufacturer should be the name of the board's creator, and Product should be the board's name and short (eg, not a description) and strings like this aren't supported, at the moment, IIRC.

Reply:
denloated